src/resolvers-cassandra/Messages/queries.js

```diff
@@ -49,64 +48,61 @@ function byBbox(args, res) { // eslint-disable-line no-unused-vars
     if (!args.bbox || args.bbox.length !== 4) return reject('Invalid bbox specified');
 
     const { fromDate, toDate } = parseFromToDate(args.fromDate, args.toDate);
+    const limit = args.limit || 15;
+    const [north, west, south, east] = args.bbox;
+
+    const tagsQuery = `
+    SELECT eventid
```

Comment:
would `distinct eventid` work for this query? Trying to mitigate the duplicate record issue.